packages/integration-karma/test/shadow-dom/ShadowRoot-properties/ShadowRoot.spec.js

```diff
@@ -84,3 +84,27 @@ describe('restrictions', () => {
         });
     }
 });
+
+if (!process.env.NATIVE_SHADOW) {
+    describe('synthetic-shadow restrictions', () => {
+        let elm;
+
+        beforeAll(() => {
+            elm = createElement('x-test', { is: Test });
+        });
+
+        it(`should throw when invoking ShadowRoot.elementsFromPoint`, () => {
+            expect(() => elm.shadowRoot.elementsFromPoint(0, 0)).toThrowErrorDev(
```

Comment:
It not only throws in dev mode but also in prod.

```suggestion
            expect(() => elm.shadowRoot.elementsFromPoint(0, 0)).toThrow(
```